api_tests/nodes/views/test_node_children_list.py

```diff
@@ -150,6 +150,54 @@ def test_node_children_list_does_not_include_node_links(
 
         assert pointed_to._id not in ids
 
+    # Regression test for https://openscience.atlassian.net/browse/EMB-593
+    # Duplicates returned in child count
+    def test_node_children_related_counts_duplicate_query_results(self, app, user, public_project,
+            private_project, public_project_url):
+        user_2 = AuthUserFactory()
+
+        # Adding a child component
+        child = NodeFactory(parent=public_project, creator=user, is_public=True, category='software')
+        child.add_contributor(user_2, ['read', 'write'], save=True)
```

Comment:
It took me awhile to figure out how to create this scenario in a test - where the count query was returning duplicates - this line added a duplicate child to the count, before the fix.  So components with multiple contributors might let you duplicate this in testing?